src/styles/page/reviews.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+
+  &__star {
+    margin: 2px;
+    height: 16px;
+    width: 16px;
+    background: url("../../images/star.svg");
+    background-repeat: no-repeat;
+  }
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url("../../images/star-active.svg");
+  background-repeat: no-repeat;
+}

```

Comment:
You only have styles for .stars--4. Consider making a loop that goes over every nth-child, like we discussed during today's lecture.